### PR TITLE
feat: insider version

### PR DESCRIPTION
**Pull request by daiyam**
This PR is preparing for the insider version with few improvements:
- separated build scripts (per platform) are regrouped as one
- simplified `prepare_vscode.sh` for updating the `product.json` properties

The keep tracks of which commit/version the patches have been built against, the `insider.json` will be used.

### CODE REVIEW

File: prepare_vscode.sh [hunk 1]
```diff
@@ -54,53 +66,86 @@ else
...
+setpath "product" "checksumFailMoreInfoUrl" "https://go.microsoft.com/fwlink/?LinkId=828886"
+setpath "product" "documentationUrl" "https://go.microsoft.com/fwlink/?LinkID=533484#vscode"
+setpath_json "product" "extensionsGallery" '{"serviceUrl": "https://open-vsx.org/vscode/gallery", "itemUrl": "https://open-vsx.org/vscode/item"}'
+setpath "product" "introductoryVideosUrl" "https://go.microsoft.com/fwlink/?linkid=832146"
+setpath "product" "keyboardShortcutsUrlLinux" "https://go.microsoft.com/fwlink/?linkid=832144"
+setpath "product" "keyboardShortcutsUrlMac" "https://go.microsoft.com/fwlink/?linkid=832143"
+setpath "product" "keyboardShortcutsUrlWin" "https://go.microsoft.com/fwlink/?linkid=832145"
+setpath "product" "licenseUrl" "https://github.com/VSCodium/vscodium/blob/master/LICENSE"
+setpath "product" "linkProtectionTrustedDomains" '["https://open-vsx.org"]'
+setpath "product" "linuxIconName" "vscodium"
```
Comment (GitMensch) [inline prepare_vscode.sh:89]: Shouldn't this be different for an insider build?
If we don't have one then just change blue to green?

**Review by GitMensch — APPROVED** [review]
LGTM

File: prepare_vscode.sh (hunk 1 above)
Comment (daiyam) [reply prepare_vscode.sh:89]: maybe for `linuxIconName`...

**Comment by daiyam** [comment]
I still need to work on the workflows...

File: prepare_vscode.sh (hunk 1 above)
Comment (daiyam) [reply prepare_vscode.sh]: I've update the property. Thx

**Comment by daiyam** [comment]
They will be out soon :wink:

The release is controlled as for the stable version, with the cron or a manual dispatch.
I will launch and test each platform one at a time.

**Comment by daiyam** [comment]
@GitMensch They are finally available at https://github.com/VSCodium/vscodium-insiders

**Pull request merged**